specification/gedcom.md

```diff
@@ -316,6 +316,20 @@ Each non-pointer payload may be encoded in 1 or more line strings (line continua
 The exact encoding of non-pointer payloads is dependent on the datatype of the payload, as determined by the structure type.
 The datatype of non-pointer payloads cannot be fully determined by line value content alone.
 
+Because empty payloads and missing payloads are considered equivalent,
+both a structure with no payload
+and a structure with the empty string as its payload
+may be encoded either
+with an empty `LineVal` (i.e., with a space after the `Tag`)
+or with no `LineVal` (i.e., without a space after the `Tag`).
+In both cases, it is recommended that the space be omitted.
```

Comment:
```suggestion
In both cases, it is recommended that the `LineVal` be omitted.
```

Reply:
Inconsistent with spec; see #91 which says it's not discouraged, it's prohibited